### PR TITLE
feat(Reveal): Add component

### LINKED ISSUE #182: <Reveal /> Component
http://semantic-ui.com/elements/reveal.html

### PR DESCRIPTION
**Pull request by layershifter**
Fixes #182

### CODE REVIEW

**Comment by Jlassos** [comment]
Thanks

**Comment by layershifter** [comment]
We have problem with augmentation on Rotate example, we cannot it use there because `Image` doesn't render children. Or I missed something?

---

Tests and docs are ready, so we need review there.

File: src/elements/Reveal/Reveal.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  children: PropTypes.node,
+
+  /** Additional classes. */
+  className: PropTypes.string,
+
+  /** A disabled reveal will not animate when hovered. */
+  disabled: PropTypes.bool,
+
+  /** An effect's name that will be applied to Reveal. */
+  effect: PropTypes.oneOf(Reveal._meta.props.effect),
```
Comment (levithomason) [inline src/elements/Reveal/Reveal.js:70]: Suggest we go with `animation` here to keep in line with the [Button `animated`](http://react.semantic-ui.com/elements/button#animated) prop.

**Review by levithomason — APPROVED** [review]
This looks great, thanks!

File: src/elements/Reveal/Reveal.js (hunk 1 above)
Comment (layershifter) [reply src/elements/Reveal/Reveal.js:70]: I'll make PR for `animated` tomorrow.

**Pull request merged**